docs/quickstart.md

```diff
@@ -0,0 +1,187 @@
+
+`GET https://example.com/cds-services` 
+
+and then configures them in the system. 
+
+![business analyst configuration](images/analyst-configuration-spec.png)
+
+This image captures a business analyst reviewing services from one CDS provider. A business analyst may review services from multiple providers and configure appropriate services per user profiles.
+
+### Invoke service on patient-view hook
+The patient-view hook is invoked when a patient chart is opened. It's one of the most basic since the logic doesn't have any prior workflow dependencies. The service called on the patient-view hook could be dependent on patient characteristics, for example: sex, problems in problems list, active medications, etc. The current version of the CDS Hooks specification allows the EHR to decide which characteristics to consider. 
+
+### Support for FHIR resources on request or prefetch
+Often a CDS service will require additional information from the EHR to perform the decision support logic, or determine the appropriate SMART app to return. Prefetch provides the EHR the capability to pass a resource when invoking a service. For example, with a patient resource included a service could do a geography search for potential environmental risk factors. Below is an example request invoked on patient-view with a patient included: 
+
+```json
+{
+   "hookInstance" : "23f1a303-991f-4118-86c5-11d99a39222e",
+   "fhirServer" : "http://hooks.smarthealthit.org:9080",
+   "hook" : "patient-view",
+   "redirect" : "http://hooks2.smarthealthit.org/service-done.html",
+   "user" : "Practitioner/example",
+   "context" : [],
+   "patient" : "1288992",
+   "prefetch" : {
+      "patientToGreet" : {
+         "response" : {
+            "status" : "200 OK"
+         },
+         "resource" : {
+            "resourceType" : "Patient",
+            "gender" : "male",
+            "birthDate" : "1925-12-23",
+            "id" : "1288992",
+            "active" : true
+         }
+      }
+   }
+}
+```
+
+
+In some cases, additional information beyond what is included in the prefetch maybe required. The CDS service can request additional information using the FHIR REST APIs:
+
+`GET [base]/AllergyIntolerance?patient=[id]`
+
+It is recommended FHIR servers implement, and CDS Services follow, locale specific implementaiton guides. In the US, the recommended implementation guides to follow are the [Argonaut Data Query Guide (DSTU2)](http://www.fhir.org/guides/argonaut/r2/) or [HL7 US Core (STU3)](http://hl7.org/fhir/us/core/index.html). Each profile page within these implementation guides includes queries FHIR servers are required to support. 
+
+### Exposed non-secured FHIR server
+A non secured FHIR server is important to support testing with a CDS service. When the EHR moves a hook so to production the system to is expected to follow the guidelines in  the [security](specification/#security) requirements.
```

Comment:
Please update the phrase "When the EHR moves a hook so to" to strike the word "so"

Reply:
Fixed in bb3b67f